js/models/languagesMd.js

```diff
@@ -199,6 +199,8 @@ module.exports = Backbone.Model.extend({
         Cases: "Cases",
         Enter: "Enter", //notTranslated
         Discover: "Discover",
+        Block: "Block", //notTranslated
+        Unblock: "Unblock", //notTranslated
```

Comment:
Yep. The //not translated tag is actually for the other languages, so when they look through them they know to translate that phrase.